lua/entities/turret_bullets2/shared.lua

```diff
@@ -116,53 +116,59 @@ function ENT:DoShot()
 			effectdata:SetScale( 1 )
 			util.Effect( "MuzzleEffect", effectdata )
 			
-		--elseif SERVER then
 			local variance = math.random( -5, 5 )
 			self:EmitSound( self.ShotSound, 50, 100 + variance )
 			self:EmitSound( "weapons/ar2/fire1.wav", 70, 60 )
 			
-			
 		end
 		
 		if IsValid(self.shootPos) and SERVER then
 			
+			print( self.Shooter )
+			

```

Comment:
should remove this  print

Reply:
i really forgot to remove that, cool, will do